examples/latest.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Accumulo Examples
+redirect_to: https://accumulo.apache.org/1.7/examples/
```

Comment:
Relative paths would be better but redirect_to doesn't prepend the site baseurl like redirect_from.  If I make it relative, it will be broken if you have baseurl when you run the site.